corehq/apps/experiments/tests/test_experiment.py

```diff
@@ -201,12 +199,7 @@ def enable_all_experiments():
 
 
 @contextmanager
-def gc_disabled():
-    enabled = gc.isenabled()
-    if enabled:
-        gc.disable()
-    try:
+def fixed_time(times):
+    times.extend(times[-1:] * 2)  # HACK extra times for prometheus_client
```

Comment:
just curious on why this was done here instead of adding the extra time in `fixed_time` function call itself?

Reply:
These are for extra `time.time()` calls that are outside the code that is being tested. Doing it here reduces duplication and makes the tests easier to understand.